Styling/README.md

```diff
@@ -413,6 +416,33 @@ Style expressions follow JavaScript conversion rules.
 
 For conversions involving vec2`, `vec3`, `vec4`, and `RegExp`, they are treated as JavaScript objects.  For example, `vec4` implicitly converts to `NaN` with `===`, `==`, `!==`, `!=`, `>`, `>=`, `<`, and `<=` operators.  In Boolean expressions, `vec2`, `vec3`, and `vec4` implicitly convert to `true`, e.g., `!!vec4(1.0) === true`.  In string expressions, `vec2`, `vec3`, and `vec4` implicitly converts to `String` using their `toString` function.
 
+### Constants
```

Comment:
I don't mind moving them to some other namespace. `Math` maybe? Or something else?